setup_env.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root" 1>&2
+   exit 1
+fi
+
+if [ "$1" == "-h" ]; then
+    echo "Argument -c allows to install git global name and email."
+    echo "Example setup_env.sh -c username usermail"
+    exit 0
+fi
+
+cp /etc/apt/sources.list /etc/apt/sources.list.backup
+
+. /etc/lsb-release
+
+packages=(
+libssl-dev
+libudev-dev
+libavahi-client-dev
+sqlite3
+libsqlite3-dev
+automake1.11
+libexpat1-dev
+ccache
+libxml2
+libxml2-dev
+git
+libpulse-dev
+subversion
+doxygen
+graphviz
+mscgen
+libbluetooth3
+libbluetooth-dev
+bluez-tools
+chromium-browser
+libgl1-mesa-dev
+libgtest-dev
+binutils
+cmake-curses-gui
+gitk
+)
+
+if [ $DISTRIB_RELEASE!="14.04" ]; then
+   sed -i -e '$a\deb http://ppa.launchpad.net/smspillaz/cmake-2.8.12/ubuntu wily main' /etc/apt/sources.list
+fi
+
+apt-get update
+apt-get install -y ${packages[@]}
+
+#if cmake not installed it will be installed. If any version of cmake installed except 2.8.12 it will be removed before installation.
+if ! dpkg -s cmake | grep "Version" | grep 2.8.12; then
+	apt-get remove -y cmake cmake-data cmake-curses-gui
```

Comment:
@AByzhynar 

@jacobkeeler and I believe that it should be documented in the README.md what the expected development environment is for setting up SDL, not forcing it on developers through a setup script.

Also this script doesn't give a notice to the developer that changes are being made to their installed packages, or even ask for permission to make these changes, which is not ok.

I also see that the script removes the currently installed versions gcc and g++ and installs it with the version specified for the SDL environment, which is even more not ok!

Any developer who wants to setup SDL Core and runs this script will mess up their development environment, and they'll have no idea why.

The setup_env script should be installing missing packages, not removing existing ones.